### ISSUE TITLE
gmtlib_get_cpt_level or gmt_save_current_cpt cause segmentation fault

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1317]
We have many failing tests today, possibly caused by commit 12da4eaf3892c0e27d05b3c1fd78dc84c0b77cad.

The error messages are:
```
234/872 Test #236: exmod/ex02.sh ...........................***Failed    0.93 sec
Set GMT_SESSION_NAME = 22877
ERROR: Caught signal number 11 (Segmentation fault) at
0   libgmt.6.dylib                      0x00000001053e832a gmtinit_get_inset_dimensions + 90
1   ???                                 0x0000000000000000 0x0 + 0
Stack backtrace:
0   libgmt.6.dylib                      0x0000000105245fc9 sig_handler + 777
1   libsystem_platform.dylib            0x00007fff6be4eb5d _sigtramp + 29
2   ???                                 0x0000000000000030 0x0 + 48
3   libgmt.6.dylib                      0x0000000105405404 gmtlib_get_cpt_level + 228
4   libgmt.6.dylib                      0x0000000105355901 gmt_save_current_cpt + 145
5   libgmt.6.dylib                      0x00000001055dbd27 GMT_makecpt + 9239
6   libgmt.6.dylib                      0x000000010527192d GMT_Call_Module + 2541
7   gmt                                 0x0000000105239651 main + 2321
8   libdyld.dylib                       0x00007fff6bc633d5 start + 1
ERROR: gmtest:54
memtrack errors: 0
exit status: 1
```

**Comment by PaulWessel** [comment]
Yes.  But fixed in my PR #1315.

**Comment by seisman** [comment]
Fixed in #1315.
